tensorzero-internal/src/endpoints/inference.rs

```diff
@@ -186,11 +186,12 @@ pub struct InferenceIds {
     name="inference",
     skip(config, http_client, clickhouse_connection_info, params),
     fields(
-        function_name = ?params.function_name,
-        model_name = ?params.model_name,
-        variant_name = ?params.variant_name,
+        function_name,
+        model_name,
+        variant_name,
         inference_id,
         episode_id,
+        otel.name = "tensorzero_function_call"
```

Comment:
We probably don't need the `tensorzero_` prefix since the service name is displayed?

![image](https://github.com/user-attachments/assets/0aa09a1f-3f50-40ad-a9b7-60dbe0b17393)

`function_inference`

(If we need prefix, does `tensorzero::` work?)